src/Events/Custom_Tables/V1/Views/V2/By_Day_View_Compatibility.php

```diff
@@ -39,22 +39,17 @@ public function prepare_day_results( array $ids = [] ) {
 			return [];
 		}
 
-		$use_site_timezone = Timezones::is_mode( 'site' );
-		$start_date_prop   = $use_site_timezone ? 'start_date_utc' : 'start_date';
-		$end_date_prop     = $use_site_timezone ? 'end_date_utc' : 'end_date';
-
 		$prepared = [];
 
 		/** @var Occurrence $occurrence */
 		foreach (
-			Occurrence::order_by( $start_date_prop, 'ASC' )
-			          ->find_all( $ids, 'post_id' ) as $occurrence
+			Occurrence::find_all( $ids, 'post_id' ) as $occurrence
```

Comment:
Why the removal of the order by? Won't this cause the prepared set of occurrences to be collected in a somewhat arbitrary manner?

Reply:
@borkweb Good question. The why, is to just help lighten the query load / optimize speed.

From following this logic, the order is completely ignored. We end up [looping on ALL of the items](https://github.com/the-events-calendar/the-events-calendar/blob/master/src/Tribe/Views/V2/Views/By_Day_View.php#L292), to create an array of "which events" show on a particular day. 

Am I missing something else? (definitely see more opportunity to optimize this in the future)